README.md

```diff
@@ -0,0 +1,11 @@
+# WCSim fiTQun Tuning Tools
+
+## Requirements
+
+- ROOT (installed with python)
+- WCSim
+- WCSimFQTuner
+- Python packages (as defined in the requirements.txt file): `python -m pip install -r requirements.txt`
```

Comment:
maybe better `conda install --yes --file requirements.txt` since we are using it? Also a comment such that that mamba/conda is highly recommended?

Reply:
fixed; we can add a comment about the mamba/conda preference too